samples/nextjs/package.json

```diff
@@ -53,28 +54,46 @@
     "@sitecore-jss/sitecore-jss-cli": "^16.1.0-canary.44",
     "@sitecore-jss/sitecore-jss-dev-tools": "^16.1.0-canary.44",
     "@sitecore-jss/sitecore-jss-manifest": "^16.1.0-canary.44",
+    "@types/chai": "^4.2.3",
+    "@types/chai-spies": "^1.0.3",
+    "@types/enzyme": "^3.10.3",
+    "@types/mocha": "^5.2.7",
     "@types/node": "^14.6.4",
     "@types/nprogress": "^0.2.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "@types/sinon": "^7.5.0",
     "@typescript-eslint/eslint-plugin": "^4.7.0",
     "@typescript-eslint/parser": "^4.7.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.4.1",
     "axios": "^0.21.1",
-    "http-proxy-middleware": "^1.2.0",
+    "chai": "^4.2.0",
+    "chai-spies": "^1.0.0",
     "chalk": "~2.4.2",
     "chokidar": "~3.1.1",
     "constant-case": "^3.0.4",
     "cross-env": "~6.0.3",
+    "enzyme": "^3.10.0",
     "eslint": "^7.13.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-yaml": "^0.2.0",
+    "graphql-import-node": "0.0.4",
     "graphql-let": "^0.16.2",
+    "http-proxy-middleware": "^1.2.0",
+    "jsdom": "15.2.1",
+    "jsdom-global": "3.0.2",
+    "mocha": "^8.1.3",
     "npm-run-all": "~4.1.5",
     "prettier": "^2.1.2",
+    "react": "^17.0.1",
```

Comment:
Why do we need it here? It's already added as prod dependency.

Reply:
As far as I know, this file is not updated manually, but by `npm i` or `npm i --save-dev`. Therefore I'm not sure how to answer this, do you see any problems with this?